### DISCUSSION TITLE
runCellQC on the output of cellranger multi

### DISCUSSION THREAD

**Discussion by esfandyari** [compbiomed/singleCellTK#742] [Q&A]
Dear SCTK team, thanks for developing such a nice and comprehensive tool!
I have used it successfully on the output of cellranger count, but I was wondering how it can be implemented for multiplexed or hybridization-based experiments, e.g., when using 10x genomics Cellplex or gene expression Flex (multiplexing biological samples in 1 capture). 
In these cases, we use cellranger Multi and the output structure is different, i.e. there are:
- only "per sample" filtered matrices, resulting from cellrenger's demultiplexing (there is no overall filtered matrix)
- "per sample" raw matrices
- an overall raw matrix containing the entire pooled sample (under multi>count>raw matrices).

Details on the output structure can be found [here]( https://www.10xgenomics.com/support/software/cell-ranger/latest/analysis/outputs/cr-flex-outputs-frp)
An example dataset can be found [here](https://www.10xgenomics.com/datasets/40k-mixture-of-mouse-cell-lines-multiplexed-samples-4-probe-barcodes-1-standard)

Do you have a recommendation for applying the pipeline for multiplexed samples? 
Does it make sense to import individual (demultiplexed per sample) filtered and raw matrices, merge them and then run the pipeline using the `sample=` argument? Or would you rather process each sample separately and then merge resulted cleaned-up objects?
When specifying the `background` in `runCellQC`, which raw data do you recommend using?

Any hints are much appreciated! Thanks!

**Comment by fstrijth** [comment]
I have the same questions, I want to run DecontX which uses the SingleCellExperiment data structure. I thought I would just try both approaches you described, but since the output structure for multiplexed data is different I can't get importCellRanger to work. If anyone has a workaround I would greatly appreciate it.
